jetty-util/src/main/java/org/eclipse/jetty/util/IncludeExcludeSet.java

```diff
@@ -98,7 +99,7 @@ public IncludeExcludeSet()
                 _excludePredicate = new SetContainsPredicate(_excludes);
             }
         }
-        catch (InstantiationException | IllegalAccessException e)
+        catch (Exception e)
```

Comment:
Add `catch (RuntimeException e)` before `catch (Exception e)` and rethrow without wrapping it.